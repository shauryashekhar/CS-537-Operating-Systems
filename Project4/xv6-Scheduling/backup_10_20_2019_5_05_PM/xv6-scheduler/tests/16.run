cd src; ../../tester/run-xv6-command.exp CPUS=1 Makefile.test test_16 | grep XV6_SCHEDULER; cd ..
