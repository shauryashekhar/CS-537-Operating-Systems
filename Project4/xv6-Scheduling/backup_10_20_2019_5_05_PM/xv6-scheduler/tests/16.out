XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 20
XV6_SCHEDULER	 	 level 0 qtail 2
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 1
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 20
XV6_SCHEDULER	 	 level 0 qtail 2
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 20
XV6_SCHEDULER	 	 level 0 qtail 2
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 40
XV6_SCHEDULER	 	 level 0 qtail 3
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 20
XV6_SCHEDULER	 	 level 0 qtail 2
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 40
XV6_SCHEDULER	 	 level 0 qtail 3
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 40
XV6_SCHEDULER	 	 level 0 qtail 3
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 60
XV6_SCHEDULER	 	 level 0 qtail 4
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 40
XV6_SCHEDULER	 	 level 0 qtail 3
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 60
XV6_SCHEDULER	 	 level 0 qtail 4
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 60
XV6_SCHEDULER	 	 level 0 qtail 4
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 80
XV6_SCHEDULER	 	 level 0 qtail 5
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 60
XV6_SCHEDULER	 	 level 0 qtail 4
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 80
XV6_SCHEDULER	 	 level 0 qtail 5
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 80
XV6_SCHEDULER	 	 level 0 qtail 5
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 100
XV6_SCHEDULER	 	 level 0 qtail 6
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 80
XV6_SCHEDULER	 	 level 0 qtail 5
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 100
XV6_SCHEDULER	 	 level 0 qtail 6
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 100
XV6_SCHEDULER	 	 level 0 qtail 6
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 120
XV6_SCHEDULER	 	 level 0 qtail 7
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 100
XV6_SCHEDULER	 	 level 0 qtail 6
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 120
XV6_SCHEDULER	 	 level 0 qtail 7
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 0
XV6_SCHEDULER	 	 level 0 ticks used 120
XV6_SCHEDULER	 	 level 0 qtail 7
